packages/core/src/types.ts

```diff
@@ -123,11 +123,12 @@ export type ProcessStats = {
 
 export type TransportOptions = {
   method: 'GET' | 'POST',
+  // @TODO: This should probably follow the fetch "HeadersInit"
+  // headers?: HeadersInit
```

Comment:
Hmm, this type is also used by the server transport. Isn't `HeadersInit` a browser type?

Reply:
Sounds good. I'll remove this, as it'd be a pain. Node 18 will be coming with native fetch + `Headers`, but thats a long ways away!